README.md

```diff
@@ -34,14 +34,26 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the difference between an adaptive website and a fully responsive website?
 
+An adaptive website adapts to devices based on some certain breakpoints in size of the viewers viewport. for instance if the viewport is resized below 800px, the website will adapt at that breakpoint.
+
+A fully responsive website will fit most screen sizes due to the way it is designed. New code is not needed for different screen sizes. The code is Universal
+
 2. Describe what it means to be mobile first vs desktop first.
 
+A mobile first website is a website designed for mobile alone in mind as opposed to desktop, and desktop first websites are website designed for to be responsive for desktop size as opposed to mobile.
+
```

Comment:
Good attempt on this. This could be improved on though, check this article for a better explanation.
[Responsive CSS "Mobile First" vs "Desktop First" Media Queries](https://www.h3xed.com/web-development/responsive-css-mobile-first-vs-desktop-first-media-queries)